scripts/run-tests.sh

```diff
@@ -6,8 +6,13 @@ export ACK_GINKGO_DEPRECATIONS=1.16.5
 export ACK_GINKGO_RC=true
 EXCLUDED_FOLDERS=""
 ALL_TESTS_FOLDERS=$(ls -d ./tests/*/)
+REPORTS_FOLDER="./reports/Debug"
 
 function run_tests {
+    if [ -d "$REPORTS_FOLDER" ]; then
```

Comment:
Please remove this code. We should handle it in the project code instead of external bash script

Reply:
this is the first script that starts running all tests, the problem is that 
we need to delete it only once, at the beginning of the code run, so we cant do it 
in one of the suites.

Reply:
done. handling it in DefineLogFile func